README.md

```diff
@@ -135,8 +144,7 @@ spec:
             - "/import/bin/import"
             - "--dimse_aet=IMPORTADAPTER"
             - "--dimse_port=2575"
-            - "--dicomweb_addr=https://healthcare.googleapis.com/v1beta1"
-            - "--dicomweb_stow_path=/projects/myproject/locations/us-central1/datasets/mydataset/dicomStores/mydicomstore/dicomWeb/studies"
+            - "--dicomweb_address=https://healthcare.googleapis.com/v1beta1/projects/myproject/locations/us-central1/datasets/mydataset/dicomStores/mydicomstore/dicomWeb"
 ```
 
 If needed, to additionally include an Export Adapter, you can add the to the
```

Comment:
The export adapter also has a split peer dicomweb addr and peer dicomweb path. I'm thinking about deprecating that too, although for this CL I think it's fine.

Reply:
For StowRsSender (sink) the parameters are called peer_dicomweb_addr and peer_dicomweb_stow_path, these can be deprecated in the same way. 
But DicomClient that is used to for wado-rs (source) gets --dicomweb_addr from config (that goes only up to api version) and the rest from incoming Pubsub message. I guess this is where addr + stow path convention originated from and where it actually makes sense.

Reply:
Agreed, can you please deprecate these as well (the peer flags) in a follow-up PR? Thanks

Reply:
Ok, in a follow-up PR.